### PR TITLE
Update 3D world

### PR DESCRIPTION
**Pull request by kxprim**

### CODE REVIEW

**Comment by rvbsanjose** [comment]
:+1:

File: client/app/explore/explore.html [hunk 1]
```diff
@@ -49,19 +67,41 @@
...
+    spotLight.shadowCameraFov = 30;
+    
+   scene.add( spotLight );
+  
+  //Other
     var light = new THREE.HemisphereLight( 0xffffbb, 0x080820, 1 );
     light.position.set( - 1, 1, - 1 );
+    light.shadowCameraVisible = true;
     scene.add( light );
+
```
Comment (scottrice10) [inline client/app/explore/explore.html:93]: Merge conflict.

File: client/app/explore/explore.html [hunk 2]
```diff
@@ -130,37 +168,68 @@
...
+    
+    var ballTexture = new THREE.Texture();
+    var imgLoader = new THREE.ImageLoader();
+		imgLoader.load( 'assets/images/PlatonicSurface_Color.jpg', function (image) {
+			ballTexture.image = image;
+			ballTexture.needsUpdate = true;
+		});
+    
+    // Add OBJ object
+    var loader = new THREE.OBJLoader();
```
Comment (scottrice10) [inline client/app/explore/explore.html:177]: Indentation off.

**Comment by scottrice10** [comment]
:+1:

**Pull request merged**